### ISSUE TITLE
My Jetpack: Apply new footer

### ISSUE THREAD

**Issue by dhasilva** [Automattic/jetpack#31598]
ref p1687874518864269-slack-C02TQF5VAJD

Follow-up to https://github.com/Automattic/jetpack/pull/30279

During an accessibility review it was noticed that My Jetpack does not use the new footer with the About, Privacy and Terms links.
Since My Jetpack can be available without the Jetpack plugin installed, it is necessary to check if the plugin is available and only show those first links if not.
